### ISSUE TITLE
Indent multiple paragraphs in visual-block mode bug.

### ISSUE THREAD

**Issue by niilohlin** [yi-editor/yi#725]
If you select more than one paragraph with a visual block. And try to indent it with ">" with vim keybindings.
```
Lorem ipsum dolor sit amet,
consectetur adipiscing elit.

Donec a diam lectus.
Sed sit amet ipsum mauris.
```
Only the first paragraph will be indented.
The result will be this:
```
    Lorem ipsum dolor sit amet,
    consectetur adipiscing elit.

Donec a diam lectus.
Sed sit amet ipsum mauris.
```

**Comment by niilohlin** [comment]
Why did you close it? The bug is different from issue #710 and is still different. This bug occurs in *visual-block* mode, not in *visual* mode.

**Comment by ethercrow** [comment]
My mistake.

**Comment by noughtmare** [comment]
This has been solved by #735, right?

**Comment by jhance** [comment]
Yea this seems to be fixed, as indicated by the tests in that PR.
